src/toil/provisioners/abstractProvisioner.py

```diff
@@ -766,6 +776,8 @@ def addToilService(self, config: InstanceConfiguration, role: str, keyPath: str
             entryPointArgs = " ".join(["'" + customDockerInitCommand + "'", entryPoint, entryPointArgs])
             entryPoint = "customDockerInit.sh"
 
+        # Set up the service. Make sure to make it default to using the
+        # actually-big temp directory.
```

Comment:
Suggestion: Might be good to include `/var/tmp` and possibly the link used earlier here (https://systemd.io/TEMPORARY_DIRECTORIES/).